sites/main-site/src/content/hackathon-projects/hackathon-march-2025/ontologies-metadata.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Working on modules and subworkflows ontology and metadata
+category: components, tooling
```

Comment:
can only be one of them

Reply:
😢 I want both

Reply:
🤔 maybe I should keep `components` to attract more people and I say there are some `tooling` issues in the description. sounds good?